### ISSUE TITLE
Using caffemodel with python caffe

### ISSUE THREAD

**Issue by hboyraz** [marvis/pytorch-caffe-darknet-convert#12]
Thank you for the code! I have used darknet2caffe to convert darknet weights to caffe then can use detect.py or caffenet.py. However if I just use python caffe and create a net using the caffemodel file the network output is not same as the caffenet.py generated output even though I validated  both networks have the same input. 
Can you please validate that the caffemodel generated by darknet2caffe can be used with python caffe?

**Comment by marvis** [comment]
Can you check whether there are bn layers and whether maxpool's size is 3

**Comment by hboyraz** [comment]
Yes there are bn layers and max pool size is 3. Basically I removed reorg and route layers from yolo.cfg file and retrained it on Coco dataset. Afterwards, I have used darknet2caffe script to convert the yolo.cfg file and weights to caffe prototxt and model files. I can used detect.py with the prototxt and caffemodel files and it works fine. However if I use caffe.Net to initialize python caffe network, the outputs are different. If you want I can share the caffemodel and prototxt files as well as the corresponding yolo files.
